.NET/Microsoft.Recognizers.Text.NumberWithUnit/Parsers/BaseCurrencyParser.cs

```diff
@@ -65,9 +65,9 @@ private static void ResolveText(List<ParseResult> prs, string source, int bias)
 
         private string GetResolutionStr(object value)
         {
-            return Config.CultureInfo != null ?
```

Comment:
This is not a breaking change. not that it is incorrect, but it should not be part of this PR.

Reply:
Done. I'll fix this file in another PR.
Thanks!